Gemfile

```diff
@@ -16,11 +16,10 @@ gem 'whenever', '~> 0.9.4', require: false
 gem 'uuid'
 
 # Gems used to provide Authentication and Authorization services.
-gem 'devise', '3.2.2'
-gem 'devise_invitable', '1.4.0'
-gem 'devise-encryptable', '0.1.1'
+gem 'devise', '~> 3.4'
```

Comment:
I think this should specify the exact version because Devise is so complicated - changes to the version we're using should explicit. Additionally, our [styleguide](https://github.com/alphagov/styleguides/blob/master/ruby.md#gemfile) says pessimistic locking should be used *when the project follows semver*, which Devise doesn't claim to, and implicitly that we should use it with the PATCH version, not the MINOR version.